src/main/java/org/rumbledb/api/Item.java

```diff
@@ -138,7 +139,7 @@ public Item compareItem(Item other, OperationalExpressionBase.Operator operator,
      * @return the list of the array members.
      */
     public List<Item> getItems() {
-        throw new RuntimeException("Item is not an array.");
+        throw new OurBadException("Item '" + this.serialize() + "' is not an array.");
```

Comment:
After my other PR is merged, you could use this.getDynamicType().toString()

Reply:
For exactly which part of the error message? ie.  'is not a .... (getDynamicType)' ?

Reply:
Instead of serializing the whole item to shorten the message. But I leave this choice up to you.

Reply:
I understand now. My goal was to provide as much clarity as possible to the user. Currently the error messaged look like this.
```
annotate(
     (
         {"id": 1, "age":  20, "weight": 68.8},
         {"id": 2, "age":  35, "weight": 72.4},
         {"id": 3, "age":  50, "weight": 76.3}
     ),
     {"id": "integer", "age": "string", "weight": "decimal"}
)
```

> ⚠️  ️Error [err: RBML0005]LINE:1:COLUMN:0:Schema error in annotate(); Error while applying the schema; Schema does not match the data;  Item '20' is not a string.

If the object causing the error is error message may be populated indeed. However, I believe this serialization could be indispensable in debugging type error oversights in a large datasets.

I was also thinking of mentioning the name of the column that the problematic data is located in. While this could be used as an alternative to the serialization of the problematic item, I still believe the serialization would be helpful in the scenerio described above.

Reply:
```
rumble$ annotate(
    (
        {"id": 1, "age":  20, "weight": 68.8},
        {"id": 2, "age":  35, "weight": 72.4},
        {"id": 3, "age":  50, "weight": 76.3}
    ),
    {"id": "integer", "age": "string", "weight": "decimal"}
)
```

> ⚠️  ️Error [err: RBML0005]LINE:1:COLUMN:0:Schema error in annotate(); Data does not fit to the given schema in field 'age';  Item '20' is not a string.\

The final look of the error messages. 
- Redundant re-throws removes
- field information for errors added
- column keyword replaced with field in error messages for consistency and hiding column concept from the user

Reply:
Yes, it makes sense. Thanks!